docs/docs/8.x/usage/roles-and-permissions.md

```diff
@@ -6,9 +6,13 @@ sidebarDepth: 2
 
 ## Setting things up
 
-Let's start by creating the following `Role`s:
+Let's start by creating the following `Roles`:
```

Comment:
Could you please put it back to `Role`s?

Reply:
Sure